Tests/timestamp_replacer.py

```diff
@@ -213,8 +213,14 @@ def clean_json_body(self, req: HTTPRequest) -> None:
                 try:
                     content = OrderedDict(literal_eval(content))
                     self.modify_json_body(req, content)
+                    return
                 except Exception:
                     logging.exception(f'failed to run literal_eval on content {content}')
+                try:
+                    content = json.loads(content, object_pairs_hook=OrderedDict)
```

Comment:
Should we perhaps add a log statement here - something to the effect of `'parsing the request body with "literal_ast" failed - trying with "json.loads"'`? What do you think?